packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -163,6 +164,14 @@ router.post(
       response,
     });
 
+    if (response.patientId && response.cxId) {
+      await processSubsequentOutboundPatientDiscoveryResps({
```

Comment:
Should we make this asynchronous? It's not relevant to IHE GW, so it doesn't need to wait and we minimize sources utilization on that server. Just gotta make sure the whole function is wrapped on a try/catch

Reply:
This has been marked as resolved and you agreed with the comment, but the code is still synchronous. Maybe forgot to push the commit to remote?

Reply:
Weird i did push - is see it in the files changed. Do you still not see it?

Reply:
I see it now 👍